### PR TITLE
SSH Passphrase prompt

### PR DESCRIPTION
**Pull request by padilo**
This PR allows to specify ssh password of your certificates, as the password.

I think it corrects #163 

Should you need to PR to master?

### CODE REVIEW

**Comment by borjavh** [comment]
I'm interested in this PR!

**Comment by ramon2** [comment]
I have the same problem and I hope this fix can be in master as soon as possible

**Comment by yacota** [comment]
I'm also very interested in this PR!

**Comment by ramonibz** [comment]
+1

File: project/build.scala [hunk 1]
```diff
@@ -6,7 +6,7 @@ object Builds extends sbt.Build {
   import sbtbuildinfo.Plugin._
   import conscript.Harness.conscriptSettings
 
-  val g8version = "0.6.8"
+  val g8version = "0.6.10-SNAPSHOT"
```
Comment (xuwei-k) [inline project/build.scala]: next version is `0.6.9-SNAPSHOT`

**Comment by xuwei-k** [comment]
Thanks!

**Pull request merged**